pom.xml

```diff
@@ -866,6 +840,140 @@
   </distributionManagement>
 
   <profiles>
+    <profile>
+      <!--
+       The 'no-jenkins-bom' profile captures the properties & dependencies when _not_ using the Jenkins BOM:
+        https://github.com/jenkinsci/jenkins/blob/master/bom/pom.xml
+       When the BOM is not being used, this profile is adding back the <dependencyManagement/> entries that were
```

Comment:
BTW use `<dependencyManagement />` since otherwise MRP will make this change for you.